api/controller/EducationController.py

```diff
@@ -124,7 +124,7 @@ def destroy_education(self, id):
                         "url": f'/api/v1/education/{id}'
                     }
                 }
-            }, 204
+            }, 202
```

Comment:
Glad we know now how to solve this red herring.

Reply:
For those of you who don't know, this is so cool!  Basically, 204 represents a no-content error...which gramatically would seem logical to put as the error response given to a delete function.  However, under the hood, python is taking this literal 204 no-content error and providing  n o   c o n t e n t.  Changing it to a 202 response allowed our coded response to be completed.